pyiode/pyiode_ws.pyx

```diff
@@ -318,8 +320,9 @@ def ws_save_var(filename: str):
 # High to Low
 # -----------
 def ws_htol(filename: str, varlist, series_type: int):
-    if isinstance(varlist, list):
-        varlist = ' '.join(varlist)
+    varlist = arg_to_str(varlist, sep = ' ')
+#    if isinstance(varlist, list):
+#        varlist = ' '.join(varlist)
```

Comment:
Please do not let dead code as comment, delete it.

Reply:
Done